scripts/prepare_motus_crates/motus_ro_crates_preparer.py

```diff
@@ -252,12 +320,12 @@ def extract_files_from_subdirectories(self, url):
 
 
 if __name__ == "__main__":
+    logging.info('feedback')
```

Comment:
```suggestion
```

Reply:
Thanks Martin. 
I included these as part of my latest commit